actian_odbc/manifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
-<connector-plugin class='actian_odbc' superclass='odbc' plugin-version='1.0.2' name='Actian Avalanche ODBC' version='18.1'>
+<connector-plugin class='actian_odbc' superclass='odbc' plugin-version='1.0.3' name='Actian Avalanche ODBC' version='18.1'>
```

Comment:
1.0.2 was not officially released. we could adopt a new release strategy - lets chat about this (the early access yesterday may be a good reason to bump.. but I personally prefer bump to be separate from other changes)

Reply:
meant to add/say and odbc/jdbc should be in the same commit when bumping moving forward

Reply:
Discussed with @hab6 both ODBC and JDBC will be bumped to 1.0.4, for this PR. This will be published and immediately we'll bump to 1.0.5 (an odd patch number) and this will indicate a dev version, the idea being that the next published version will be even (likely 1.0.6).